src/js/background.js

```diff
@@ -124,6 +124,15 @@ function Badger(from_qunit) {
     console.log("Privacy Badger is ready to rock!");
     self.INITIALIZED = true;
 
+    if (self.criticalError == "Privacy Badger failed to initialize") {
+      delete self.criticalError;
+      chrome.tabs.query({ active: true, lastFocusedWindow: true }, (tabs) => {
+        if (tabs[0]) {
+          self.updateBadge(tabs[0].id);
+        }
+      });
+    }
```

Comment:
We set "failed to initialize" when a content script or PB's popup or options want something, and `badger.INITIALIZED` is still false ten seconds after PB's background process started running.

If PB in fact does finish initializing, PB didn't fail to initialize, it was just really slow. We should unset "failed to initialize" at this point, right? Not sure what other condition we should check before deleting the error.

We could take out all of "failed to initialize" logic, but then we we would back at not knowing if PB were to ever fail to initialize again. I think not knowing is quite bad because then we're silently failing some proportion of our users.

Reply:
That makes sense, so do we not have to check any condition to confirm that PB actually initialized because this code won't run unless PB finished initializing?

Reply:
Yeah, if we got to this far, `badger.INITIALIZED` is `true`.